modules/fb_instant_articles_display/src/InstantArticle.php

```diff
@@ -69,19 +68,20 @@ class DrupalInstantArticleDisplay {
    * @param $instantArticle
    */
   private function __construct($node, $layoutSettings, $instantArticle) {
+    parent::__construct();
```

Comment:
Parent constructor is private, so this fatals.  I think we can leave this out.  This constructor is also private, meant to be created via `create()`, which calls the `InstantArticle::create()` method, which in turn calls the private constructor for that class, so this is unnecessary.